packages/atomic/src/components/commerce/facets/atomic-commerce-facets/atomic-commerce-facets.tsx

```diff
@@ -35,6 +36,7 @@ import {CommerceBindings as Bindings} from '../../atomic-commerce-interface/atom
 export class AtomicCommerceFacets implements InitializableComponent<Bindings> {
   @InitializeBindings() public bindings!: Bindings;
   public facetGenerator!: FacetGenerator;
+  public summary!: ListingSummary | SearchSummary;
```

Comment:
Just the heads up, ListingSummary / SearchSummary will soon become sub-controllers:

https://github.com/coveo/ui-kit/pull/4082